calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx

```diff
@@ -134,7 +134,7 @@ watch kubectl get tigerastatus
 
 **Recommended - Networking**
 
-- The default networking uses IP-in-IP with BPG routing. For all networking options, see [Determine best networking option](../../networking/determine-best-networking.mdx).
+- The default networking uses IP-in-IP with BGProuting. For all networking options, see [Determine best networking option](../../networking/determine-best-networking.mdx).
```

Comment:
@samkookji77 I see that you force-pushed some changes after my last comment. But as you can see, the change didn't get into the revision. Can you try again?

Reply:
Hi, @ctauchen Do you want to change from 'BGProuting' to 'BGP routing'?

Reply:
@ctauchen Thank you for your reply. I fixed that typo and push it.